src/store/useStore.js

```diff
@@ -0,0 +1,9 @@
+const store = require('./store');
+
+module.exports = function (app) {
+  app.set('store', store);
+  app.use(function (req, res, next) {
+    req.store = store;
+    next();
+  });
```

Comment:
1. I don't think you're using `app.get('store')` anywhere, are you?
2. Remove this file and move the store middleware to `app.js`